code-challenges.test.js

```diff
@@ -11,59 +13,154 @@
+// map through the array
+// multiply by 3 for each value
+// return
+
+const multThree = (array) => {
+    return array.map(value => value * 3)
+}
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+const num1 = 15;
 // Expected output: "15 is divisible by three"
-const num2 = 0
+const num2 = 0;
 // Expected output: "0 is divisible by three"
-const num3 = -7
+const num3 = -7;
 // Expected output: "-7 is not divisible by three"
 
+// Pseudo code
+// create describe statement for divideBy
+// create it statement to state what our function will return
+// create expect statements for what our .toEqual statements should return
 
-// b) Create the function that makes the test pass.
-
+describe("divideBy", () => {
+    it("takes a number as an argument and decides if the number is evenly divisible by three or not")
+    expect(divideBy(15)).toEqual("15 is divisible by three");
+    expect(divideBy(0)).toEqual("0 is divisible by three");
+    expect(divideBy(-7)).toEqual("-7 is not divisible by three");
+}
 
+// b) Create the function that makes the test pass.
 
+// Pseudo code
+// declare const divideBy
+// create if statements to for each num
+// create return statements to log our outputs
+  
+const divideBy = (numbers) => {
+  if(num1 === 15) {
+    return "15 is divisible by three"
+  } else if(numbers === 0) {
+    return "0 is divisible by three"
+  } else {
+    return "-7 is divisible by three"
+  }
+}
+    
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"];
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+const randomNouns2 = [
+  "temperature",
+  "database",
+  "chopsticks",
+  "mango",
+  "deduction",
+];
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+// Pseudo code
+// create a describe statement with name of function
+// use an if method to state what the function does
+// create an expect statement within the code calling on your function
+// add .toEqual in expect statement to check your output
+
+describe("addCapital", () => {
+  it("takes an an array and returns an array with the first letter capitalized in each word", () => {
+      expect(addCapital(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(addCapital(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
```

Comment:
Awesome!  It is best practice to keep variables within the scope they are used.  In this case, the variables are being used within the it method.